### PR TITLE
Added docker image and kubernetes files

### PR DESCRIPTION
**Pull request by w0mbat**
I added a Dockerfile, Kubernetes deployment/service and build instructions

### CODE REVIEW

**Comment by brancz** [comment]
I like the general idea, just wondering if we can get the image into gcr under the `google-containers` namespace? Additionally it would be nice to build this via a pipeline on releases. @fabxc

**Comment by w0mbat** [comment]
@brancz @fabxc Sure, just wanted to make it as easy as possible for everyone to try it out first before some build, tags and pushes it to an official location - I will be happy to update the PR once I get an image location I can link to :)

**Comment by brancz** [comment]
I'm in line here with @fabxc, I'm fine with merging this with a disclaimer in the README and the fixes. My comment was generally me wondering if there is (there probably is) a process or something to have an image on gcr and or build process for it.

**Comment by w0mbat** [comment]
@brancz @fabxc All good now?

File: README.md [hunk 1]
```diff
@@ -90,11 +90,27 @@ for metric collection at all and instead implement their own. Having
...
 # Usage
 
 Simply build and run kube-state-metrics inside a Kubernetes pod which has a
 service account token that has read-only access to the Kubernetes cluster.
 
+## Kubernetes Deployment
+
+DISCLAIMER: At the moment the Docker image is hosted under a private account until we managed to push it to an official google-containers repo and integrate it with the release chain.
+
+To deploy this project, you can simply run `kubectl apply -f kubernetes` and a Kubernetes service and deployment will be created. The service already has a `prometheus.io/scrape: 'true'` annotation and if you added the recommended Prometheus service-endpoint scraping configuration, Prometheus will pick it up automatically and you can start using the generated metrics right away.
```
Comment (brancz) [inline README.md]: maybe add a link about "the recommended Prometheus service-endpoint scraping configuration"

Comment (brancz) [inline README.md]: And just my OCD, but I like to do 80 char line breaks, but that's purely preference, feel free to ignore.

Comment (w0mbat) [inline README.md]: @brancz here you go :)

**Comment by brancz** [comment]
LGTM 👍 . I'll give @fabxc a chance to re-review. :)

**Comment by fabxc** [comment]
Thanks

**Pull request merged**